_extensions/grantmcdermott/clean/_extension.yml

```diff
@@ -0,0 +1,13 @@
+title: clean
```

Comment:
If these files in _extensions are enough to install the extension, then great.  Otherwise, do these want to be in the repo, if they're the same as get installed when the extension gets installed.  If these are enough, the github action should be changed to not install the extension.

Reply:
Not in or in? Comment says not in, but you've added them in.